src/tiled/tileseteditor.cpp

```diff
@@ -417,6 +418,14 @@ void TilesetEditor::retranslateUi()
     mEditTerrain->setText(tr("Edit &Terrain Information"));
 }
 
+bool hasTileInTileset(QString tileName, const QMap<int, Tile*> addedTiles){
```

Comment:
Please mark this function as `static` since it's not used from other modules.

Coding style: please put the `{` on a new line (for functions).

Naming: `tileName` is confusing, please call it `imageSource` instead.
Naming: `addedTiles` are not actually the added tiles, they're the tiles already in the tileset. Please pass in `const Tileset &tileset` instead.